lib/Capabilities.php

```diff
@@ -184,7 +184,7 @@ public function __construct(
 	/**
 	 * @return array{
 	 *      spreed: TalkCapabilities,
-	 * }|array<empty>
+	 * }|array<string, empty>
```

Comment:
This is not parsed well by openapi-extractor, I think the better approach is to use `spreed?: TalkCapabilities`.

Reply:
but `array{spreed?: …}` can be an empty array, which is then a list and later on fails, but I can't do the stdClass cheat there as the capabilities interface only allows arrays to be returned.
But yeah I noticed this mistake already as well and am thinking of a solution.

Reply:
It doesn't matter though, as the empty case is handled by the server. The annotation is also valid for `return [];` so it is fine.

Reply:
It's handled by the server but not by the room/v4/.../capabilities endpoint. But yeah I will cheat then